deployment/GI03FLMA_Deploy.csv

```diff
@@ -129,3 +129,29 @@ AR30-03,,,,GI03FLMA-RIS01-00-SIOENG000,5,1,2018-06-10T13:59:00,,CGMGI-03FLMA-000
 AR30-03,,,,GI03FLMA-RIS01-03-DOSTAD000,5,1,2018-06-10T13:59:00,,CGMGI-03FLMA-00005,,CGINS-DOSTAD-00391,59.76805,-39.84252,,30,2700,
 AR30-03,,,,GI03FLMA-RIS01-04-PHSENF000,5,1,2018-06-10T13:59:00,,CGMGI-03FLMA-00005,,CGINS-PHSENF-P0182,59.76805,-39.84252,,30,2700,
 AR30-03,,,,GI03FLMA-RIS01-05-FLORTD000,5,1,2018-06-10T13:59:00,,CGMGI-03FLMA-00005,,CGINS-FLORTD-01319,59.76805,-39.84252,,30,2700,
+AR35-05,,,,GI03FLMA-RI000-00-CTDMOH000,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-12218,,,,1700,2689,
+AR35-05,,,,GI03FLMA-RI000-00-CTDMOH100,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-12619,,,,2600,2689,
+AR35-05,,,,GI03FLMA-RI000-00-CTDMOH400,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-12395,,,,2300,2689,
+AR35-05,,,,GI03FLMA-RI000-00-CTDMOH700,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-12387,,,,2000,2689,
+AR35-05,,,,GI03FLMA-RI000-00-VELPTB000,6,1,,,CGMGI-03FLMA-00006,,CGINS-VELPTB-11970,,,,1700,2689,
+AR35-05,,,,GI03FLMA-RI000-00-VELPTB100,6,1,,,CGMGI-03FLMA-00006,,CGINS-VELPTB-12197,,,,2600,2689,
+AR35-05,,,,GI03FLMA-RI000-00-VELPTB400,6,1,,,CGMGI-03FLMA-00006,,CGINS-VELPTB-12095,,,,2300,2689,
+AR35-05,,,,GI03FLMA-RI000-00-VELPTB700,6,1,,,CGMGI-03FLMA-00006,,CGINS-VELPTB-11981,,,,2000,2689,
+AR35-05,,,,GI03FLMA-RIM01-00-SIOENG000,6,1,,,CGMGI-03FLMA-00006,,CGCON-SIOCON-13994,,,,1507,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-ADCPSL003,6,1,,,CGMGI-03FLMA-00006,,CGINS-ADCPSL-21450,,,,500,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG040,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-11643,,,,30,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG041,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-12401,,,,40,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG042,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-11638,,,,60,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG043,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10222,,,,90,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG044,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-11595,,,,130,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG045,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10253,,,,180,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG046,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10225,,,,250,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG047,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10223,,,,350,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOG048,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOG-10256,,,,500,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH049,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-10265,,,,750,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH050,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-10227,,,,1000,2689,
+AR35-05,,,,GI03FLMA-RIM01-02-CTDMOH051,6,1,,,CGMGI-03FLMA-00006,,CGINS-CTDMOH-11597,,,,1500,2689,
+AR35-05,,,,GI03FLMA-RIS01-00-SIOENG000,6,1,,,CGMGI-03FLMA-00006,,CGCON-SIOCON-13992,,,,30,2689,
+AR35-05,,,,GI03FLMA-RIS01-03-DOSTAD000,6,1,,,CGMGI-03FLMA-00006,,CGINS-DOSTAD-00515,,,,30,2689,
+AR35-05,,,,GI03FLMA-RIS01-04-PHSENF000,6,1,,,CGMGI-03FLMA-00006,,CGINS-PHSENF-P0141,,,,30,2689,
+AR35-05,,,,GI03FLMA-RIS01-05-FLORTD000,6,1,,,CGMGI-03FLMA-00006,,CGINS-FLORTD-01212,,,,30,2689,

```

Comment:
Looks good.  Water depth will need to be updated after deployment, and deployment depth of the OSNAP instruments which should be 100, 400, 700, and 1000 less than the water depth.